app/helpers/argo_helper.rb

```diff
@@ -147,7 +147,7 @@ def render_buttons(doc, object = nil)
       # date=embargo_data.split.last
       if text != 'released'
         # TODO: add a date picker and button to change the embargo date for those who should be able to.
-        buttons << {:label => 'Update embargo', :url => embargo_form_item_path(pid)} if current_user.is_admin
+        buttons << {:label => 'Update embargo', :url => embargo_form_item_path(pid)} if can?(:manage_embargo, object)
```

Comment:
`cancan` offers us an opportunity to clearly separate attributes of the user (like group membership) from the actions they can perform. The underlying behavior isn't changed, we're simply decoupling user attributes from the view.